core-js/src/main/javascript/cdf/dashboard/Utils.js

```diff
@@ -734,6 +734,52 @@ define([
     return "rgb(" + rgb.join(",") + ")";
   };
 
+  /**
+   * Determines if `path` ends with `action`,
+   * immediately preceded by a `\` or `/` separator character.
+   *
+   * Pattern to test: `<path> === <basepath>[\/]<action>`.
+   *
+   * This implementation, in particular, strives to not allocate memory.
+   *
+   * @param {*} path - The path.
+   * @param {*} action - The action.
+   * @return {boolean} `true` if yes; `false`, otherwise.
+   */
+  Utils.pathIncludesAction = function (path, action) {
+
+    if(typeof path !== "string" || typeof action !== "string") {
+      return false;
+    }
+
+    if(action.length >= path.length) {
+      return false;
+    }
+
+    if( "\\/".indexOf(path.substr(-1, 1)) >= 0) {
```

Comment:
This is a special case that would otherwise be immediately detected by `endsWith` (assuming `action` does not itself end with a `/` or `\`). So why repeat it?

Reply:
This condition is exactly to prevent actions that finish with `/` or `\`. 

I found this when i was running these tests:
```
expect( Utils.pathIncludesAction( "/a.ext/", "a.ext/" ) ).toBe( false );
expect( Utils.pathIncludesAction( "/a.ext\\", "a.ext\\" ) ).toBe( false );
```